how-to-docs/amazon-resources.md

```diff
@@ -0,0 +1,7 @@
+# How to get Amazon Resources in the UK Cluster
+
+1. Clone the [stacks-hod-platform](https://gitlab.digital.homeoffice.gov.uk/Devops/stacks-hod-platform/) repo on gitlab
```

Comment:
I think this is the old one; we need the new ACP ones. Also, I believe there are separate repos for different environments (notprod vs prod being the ones that matter to our users).

@KashifSaadat may be able to provide the definitive list of repos.

Also, I recommend looking at the old docs, as a starting point.

Reply:
Hey. So this How-To doc should probably be renamed, as it's requests for resources in the UK accounts, not within the kube clusters themselves.

For now we will be re-using the `stacks-hod-platform` repo I believe, but in the future likely to change / convert to something else (whether it's use of terraform and standardised acp modules, or some other tool).